datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaConnector.java

```diff
@@ -73,6 +77,11 @@
   private static final Duration SHUTDOWN_EXECUTOR_SHUTDOWN_TIMEOUT = Duration.ofSeconds(30);
   static final Duration MIN_DAEMON_THREAD_STARTUP_DELAY = Duration.ofMinutes(2);
 
+  private static final String MODULE = AbstractKafkaConnector.class.getSimpleName();
```

Comment:
Since this is an abstract class which needs to be extended by actual Connectors, can we take a metrics prefix as input and use that to construct the metrics instead? This way, the metrics will be more meaningful, and it'll be easier to differentiate between different connectors that use this as their base.

You can look at `AbstractKafkaBasedConnectorTask` to see how connector tasks that extend it construct and pass their metrics prefix. We do want both the extending class name and the connector name to be part of the metrics prefix. This is because we have many connectors which use the same class.

Reply:
Done.